dpnp/dpnp_iface_mathematical.py

```diff
@@ -1605,16 +1606,27 @@ def subtract(x1,
     return call_origin(numpy.subtract, x1, x2, out=out, where=where, dtype=dtype, subok=subok, **kwargs)
 
 
-def sum(x1, axis=None, dtype=None, out=None, keepdims=False, initial=None, where=True):
+def sum(x, /, axis=None, dtype=None, keepdims=False, *, out=None, initial=False, where=True):
     """
     Sum of array elements over a given axis.
 
     For full documentation refer to :obj:`numpy.sum`.
 
+    Returns
+    -------
+        y : dpnp.ndarray
+            an array containing the sums. If the sum was computed over the
+            entire array, a zero-dimensional array is returned. The returned
+            array has the data type as described in the `dtype` parameter
+            description above.
```

Comment:
I guess that needs to be rephrased, since there is no above description of `dtype`